[2022-07-04 23:13:20,440: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 23:13:20,440: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 23:13:20,442: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 23:13:20,443: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 23:13:20,443: INFO: common]: created directory at: artifacts\prepared
[2022-07-04 23:13:20,443: ERROR: stage_01_prepare_data]: [Errno 2] No such file or directory: 'data\\data.xml'
Traceback (most recent call last):
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 62, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 44, in main
    with open(source_data_path, encoding = encode) as fd_in: #actual input data that we are reading
FileNotFoundError: [Errno 2] No such file or directory: 'data\\data.xml'
[2022-07-04 23:18:41,902: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 23:18:41,902: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 23:18:41,905: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 23:18:41,905: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 23:18:41,905: INFO: common]: created directory at: artifacts\prepared
[2022-07-04 23:18:41,906: ERROR: stage_01_prepare_data]: name 'target_tag' is not defined
Traceback (most recent call last):
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 62, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 47, in main
    process_posts(fd_in, fd_out_train, fd_out_test, target_tag, split)
NameError: name 'target_tag' is not defined
[2022-07-04 23:39:52,429: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 23:39:52,429: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 23:39:52,431: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 23:39:52,431: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 23:39:52,432: INFO: common]: created directory at: artifacts\prepared
[2022-07-04 23:39:52,432: ERROR: stage_01_prepare_data]: name 'target_tag' is not defined
Traceback (most recent call last):
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 62, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 47, in main
    process_posts(fd_in, fd_out_train, fd_out_test, target_tag, split)
NameError: name 'target_tag' is not defined
[2022-07-04 23:44:12,192: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 23:44:12,192: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 23:44:12,195: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 23:44:12,195: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 23:44:12,196: INFO: common]: created directory at: artifacts\prepared
[2022-07-04 23:44:12,196: ERROR: stage_01_prepare_data]: name 'target_tag' is not defined
Traceback (most recent call last):
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 62, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 47, in main
    process_posts(fd_in, fd_out_train, fd_out_test, target_tag, split)
NameError: name 'target_tag' is not defined
[2022-07-05 12:44:41,165: INFO: stage_01_prepare_data]: 
********************
[2022-07-05 12:44:41,166: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-05 12:44:41,218: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-05 12:44:41,219: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-05 12:44:41,219: INFO: common]: created directory at: artifacts\prepared
[2022-07-05 12:44:42,329: ERROR: stage_01_prepare_data]: name 'target_tag' is not defined
Traceback (most recent call last):
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 62, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 47, in main
    process_posts(fd_in, fd_out_train, fd_out_test, target_tag, split)
NameError: name 'target_tag' is not defined
[2022-07-05 13:15:42,681: INFO: stage_01_prepare_data]: 
********************
[2022-07-05 13:15:42,681: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-05 13:15:42,683: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-05 13:15:42,683: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-05 13:15:42,683: INFO: common]: created directory at: artifacts\prepared
[2022-07-05 13:15:42,684: ERROR: stage_01_prepare_data]: name 'target_tag' is not defined
Traceback (most recent call last):
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 62, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 47, in main
    process_posts(fd_in, fd_out_train, fd_out_test, target_tag, split)
NameError: name 'target_tag' is not defined
[2022-07-05 13:21:42,752: INFO: stage_01_prepare_data]: 
********************
[2022-07-05 13:21:42,752: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-05 13:21:42,754: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-05 13:21:42,755: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-05 13:21:42,755: INFO: common]: created directory at: artifacts\prepared
[2022-07-05 13:21:42,755: ERROR: stage_01_prepare_data]: name 'target_tag' is not defined
Traceback (most recent call last):
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 62, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "D:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\stage_01_prepare_data.py", line 47, in main
    process_posts(fd_in, fd_out_train, fd_out_test, target_tag, split)
NameError: name 'target_tag' is not defined
